src/ts/app.ts

```diff
@@ -0,0 +1,91 @@
+import Equation from "./equation.js";
+
+export default class App {
+  totalDiv: HTMLDivElement;
+  constructor() {
+    const equation = new Equation();
+    this.totalDiv = document.querySelector("#total") as HTMLDivElement;
+    document
+      .querySelector("div.modifiers.subgrid")!
+      .addEventListener("click", () => this.allClear(equation));
+    document
+      .querySelector("div.digits.flex")!
+      .addEventListener("click", evt =>
+        this.typeNumber(evt.target as HTMLButtonElement, equation)
+      );
+    document
+      .querySelector("div.operations.subgrid")!
+      .addEventListener("click", evt =>
+        this.clickOperations(evt.target as HTMLButtonElement, equation)
+      );
+  }
+  allClear(equation: Equation) {
+    equation.initNumbers();
+    this.totalDiv.innerText = "0";
+  }
+  typeNumber(target: HTMLButtonElement, equation: Equation) {
+    if (target.className !== "digit") {
+      return;
+    }
+    if (isNaN(Number(this.totalDiv.innerText))) {
+      this.totalDiv.innerText = "0";
+    }
+    if (equation.getOperation() === "=") {
+      this.totalDiv.innerText = "0";
+      equation.setOperation("");
+    }
+    if (this.totalDiv.innerText.length < 3) {
+      this.totalDiv.insertAdjacentText("beforeend", target.innerText);
+      this.totalDiv.innerText = String(Number(this.totalDiv.innerText));
+    }
+  }
+  clickOperations(target: HTMLButtonElement, equation: Equation) {
+    if (target.className !== "operation") {
+      return;
+    }
+    const operation: string = target.innerText;
+    const num = Number(this.totalDiv.innerText);
```

Comment:
```suggestion
    const num: number = Number(this.totalDiv.innerText);
```

저는 적는게 좋다고 생각하는데 혹시 어떻게 생각하시나요?

Reply:
그게 맞는 것 같습니다. 타입스크립트 쓰는 김에 타입을 적을 수 있는 부분은 모두 적는게 좋은 것 같아요.